cli/cli_test.go

```diff
@@ -450,79 +450,57 @@ func Example_zone() {
 	c := newCLITest()
 	defer c.stop()
 
-	zone100 := `replicas:
-- attrs: [us-east-1a,ssd]
-- attrs: [us-east-1b,ssd]
-- attrs: [us-west-1b,ssd]
-range_min_bytes: 8388608
-range_max_bytes: 67108864
-`
+	const zone1 = `replicas:
+- attrs: [us-east-1a,ssd]`
+	const zone2 = `range_max_bytes: 134217728`
+
 	c.Run("zone ls")
 	// Call RunWithArgs to bypass the "split-by-whitespace" arg builder.
-	c.RunWithArgs([]string{"zone", "set", "100", zone100})
+	c.RunWithArgs([]string{"zone", "set", "system", zone1})
 	c.Run("zone ls")
-	c.Run("zone get 100")
-	c.Run("zone rm 100")
+	c.Run("zone get system.nonexistent")
+	c.Run("zone get system.lease")
+	c.RunWithArgs([]string{"zone", "set", "system", zone2})
+	c.Run("zone get system")
+	c.Run("zone rm system")
 	c.Run("zone ls")
+	c.Run("zone rm .default")
```

Comment:
can you add a case to modify `.default`. it looks fine since it should always exist so the `UPDATE` vs `INSERT` check should always say `UPDATE`, but the use of `0` as a return value for `queryZonePath` makes it worth checking.